mmv1/products/gkehub2/Namespace.yaml

```diff
@@ -0,0 +1,125 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Namespace'
+base_url: 'projects/{{project}}/locations/global/scopes/{{scope_id}}/namespaces'
```

Comment:
Is this the correct URL location? The API documentation I can find at https://cloud.google.com/anthos/fleet-management/docs/reference/rest/v1beta/projects.locations.namespaces seems to be referencing this without the scopes sub directory.